tslint.json

```diff
@@ -5,7 +5,7 @@
     "tslint-no-unused-expression-chai"
   ],
   "rules": {
-    "indent": [ true, "tabs" ],
+    "indent": [ false, "tabs" ],
```

Comment:
NIT: It is probably a good idea to move to `resin-lint` instead, but it doesn't have to be part of this PR if you don't have the time.